server/migrations/versions/c78e97acf44d_.py

```
@@ -0,0 +1,122 @@
+    sa.Column('password_hash', sa.String(length=128), nullable=False),
+    sa.Column('role', sa.String(length=20), nullable=False),
+    sa.PrimaryKeyConstraint('id'),
+    sa.UniqueConstraint('username')
+    )
+    op.create_table('bookings',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('booking_date', sa.DateTime(), nullable=False),
+    sa.Column('appointment_date', sa.DateTime(), nullable=False),
+    sa.Column('status', sa.String(length=20), nullable=True),
+    sa.Column('notes', sa.String(), nullable=True),
+    sa.Column('clinic_id', sa.Integer(), nullable=True),
```

Flake8 (7.2.0)

[error] continuation line missing indentation or outdented

(E122)